server/controllers/profile.js

```diff
@@ -38,4 +36,35 @@ exports.loadProfile = asyncHandler(async (req, res, next) => {
       profile: profile,
     },
   });
+});
+
+// @route GET /profile/sitter
+// @desc Get all profile data
+// @access Private
+exports.getAllSitters = asyncHandler(async (req, res) => {
+  const result = [];
+  const profiles = await Profile.find({ isSitter: true, price: { $ne: null } })
+  if (profiles) {
```

Comment:
I don't see any need for this check since an array will always be truthy and the mongoose `find` query returns an array.

Reply:
Removed the if clause